services/logging.js

```diff
@@ -0,0 +1,12 @@
+require('express-async-errors');
+
+module.exports = function() {
+  process.on('uncaughtException', error => {
+    console.log(error);
+    process.exit(1);
+  });
```

Comment:
uncaughtException is going to be deprecated soon. (also it doesn't give complete log info)

Reply:
It isn't recommended to resume the app on uncaughtException. Check [here](https://nodejs.org/api/process.html#process_warning_using_uncaughtexception_correctlyl)

Reply:
okay.